pkg/resource/github/github _branch_protection_ext.go

```diff
@@ -0,0 +1,42 @@
+package github
+
+import (
+	"encoding/base64"
+	"fmt"
+
+	"github.com/cloudskiff/driftctl/pkg/resource"
+)
+
+func (r *GithubBranchProtection) String() string {
+	if r.Pattern != nil {
+		repoId := ""
+		if r.RepositoryId != nil {
+			decodedId, err := base64.StdEncoding.DecodeString(*r.RepositoryId)
+			if err == nil {
+				repoId = string(decodedId)
+			}
+		}
+
+		if repoId == "" {
+			return fmt.Sprintf("Branch: %s (Id: %s)", *r.Pattern, r.Id)
+		}
+		return fmt.Sprintf("Branch: %s, RepoId: %s", *r.Pattern, repoId)
```

Comment:
Isn't missing parenthesis for RepoId ?

Reply:
This is a good question, WDYT @moadibfr @wbeuil ? Should we display `Branch: %s (RepoId: %s)` to be more consistent with `Branch: %s (Id: %s)` ?

Reply:
Let's be consistent then. But I'm fine with one or the other

Reply:
Fixed